### ISSUE TITLE
Find a better way to handle tuple tensor inputs

### ISSUE THREAD

**Issue by RyanBaten** [RyanBaten/pytorch-diagrams#31]
LSTM requires (h_0, c_0) input and (h_n, c_n) output.
This would maybe be best split out into three ports but the compiler currently doesn't support this.

**Comment by RyanBaten** [comment]
Along the same grain, torch.cat requires its first input to be a tuple and the second arg is dim.
